tests/test_stoiip.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+from stoiip_prob import stoiip as stp
+
+
+def test_stoiip():
```

Comment:
Here we need to add a brief documentation on what is the purpose of this test. You can use the triple quotes with a single message (no need to use all the template that is fill out automatically by PyCharm in this case)

Reply:
Done. 

I activated the docstrings with the NumPy configuration (see screenshot), however, it seems like it is not working in my Pycharm.

Reply:
![image](https://user-images.githubusercontent.com/57779200/163884406-125141bf-5df5-44f5-b679-60ff43be853d.png)